src/components/DayPickerRangeController.jsx

```diff
@@ -107,6 +107,7 @@ export default class DayPickerRangeController extends React.Component {
     };
 
     this.isTouchDevice = isTouchDevice();
+    this.today = moment();
```

Comment:
I think just updating this value in `componentWillUpdate` will accomplish the same thing and create slightly fewer new moment objects... isToday is going to be called potentially hundreds of times per render.